### DISCUSSION TITLE
皮肤站的uuid生成配置是不是没了？

### DISCUSSION THREAD

**Discussion by ZHI-CCC** [bs-community/blessing-skin-server#582] [Q&A] [answered]
今天我想改生成算法
结果翻来覆去找不到了？？？
是不是删了

**Comment by tnqzh123** [comment, marked as answer]
没有。

https://github.com/bs-community/blessing-skin-plugins/blob/54ac0f31beb98f2ffebe73d517dfa64ce3c75dd1/plugins/yggdrasil-api/src/Controllers/ConfigController.php#L20-L23
